pyproject.toml

```diff
@@ -73,7 +74,6 @@ filterwarnings = [
     "ignore:.*Field .* has conflict with protected namespace \"model_\".*:UserWarning",
 ]
 asyncio_mode = "auto"
-py311_task = true
```

Comment:
this is a toggle i created to activate the patch that fixes everything, so that behavior has to be explicitly opted into.  you need to include it.